Src/BootCamp.Chapter/CleanTable.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class CleanTable
+    {
+        private const char DollarSign = '£';
+        private string _corruptedBalanceFile;
+        private string _fixedBalanceFile;
+
+        public CleanTable(string corruptedBalanceFile, string fixedBalanceFile)
+        {
+            if (!File.Exists(corruptedBalanceFile))
+            {
+                throw new ArgumentException($"The corrupted balance file does not exist: {corruptedBalanceFile}.");
+            }
+            _corruptedBalanceFile = corruptedBalanceFile;
+
+            if (File.Exists(fixedBalanceFile))
+            {
+                File.Delete(fixedBalanceFile);
+            }
+            _fixedBalanceFile = fixedBalanceFile;
+        }
+
+        public void Clean()
+        {
+            var corruptedBalanceLines = File.ReadAllLines(_corruptedBalanceFile);
+            var fixedBalanceLines = new string[corruptedBalanceLines.Length];
+            FixCorruptedBalanceFile(corruptedBalanceLines, fixedBalanceLines);
+
+            DisplayStasticalData(CreateBalanceLinesForTextTable(fixedBalanceLines));
+        }
+
+        private string[] CreateBalanceLinesForTextTable(string[] fixedBalanceLines)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'CreateBalanceLinesForTextTable' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5063788)